temporian/test/api_test.py

```diff
@@ -55,9 +48,31 @@ def disabled_test_create_processor(self):
                 a: input_signal_data,
             },
         )
-
         logging.info("results: %s", results)
 
+    def test_serialization(self):
+        a = t.place_holder(
```

Comment:
Is the placeholder op still necessary? In [this test](https://github.com/google/temporian/blob/main/temporian/test/prototype_test.py#L52) we create an `Event` from scratch that is then passed to some operators. Is making this an operator necessary? I recall also talking about the pro of allowing the user to input any intermediate event as `input` when calling `evaluate`if the input events are not "special" in any way, rather just normal events without a `creator`.

Reply:
You right, the placeholder op is not necessary. We can replace it with a helper function with an equivalent signature or let the users create the events manually. Let's do some test / chat if an helper function make sense.

*Added todo in place_holder.py.